packages/react/src/list/ListItem/ListItem.module.css

```diff
@@ -13,7 +13,7 @@
 }
 
 .ListItem__leading-visual {
-  margin: var(--base-size-4) 0;
+  height: 1lh;
```

Comment:
This change ensures that the visuals and text of list items are vertically aligned, even when the text size is overridden, as we do in PricingOptions.